.ci.yaml

```diff
@@ -2454,6 +2454,29 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Mac entrypoint_dart_registrant
+    recipe: devicelab/devicelab_drone
+    timeout: 60
```

Comment:
Per the frameworks flaky test policy, you need to add bringup: true to not block the tree. The bot will then mark this as blocking once it has enough data.

You also need to add a TESTOWNERS line

Reply:
done